statsprinter.go

```diff
@@ -159,44 +213,83 @@ func (p *planePrinter) printStatistics(t tcping) {
 }
 
 func (p *planePrinter) printProbeSuccess(hostname, ip string, port uint16, streak uint, rtt float32) {
+	var message string
 	if hostname == "" {
-		colorLightGreen("Reply from %s on port %d TCP_conn=%d time=%.3f ms\n",
-			ip, port, streak, rtt)
-		return
+		message = fmt.Sprintf("Reply from %s on port %d TCP_conn=%d\n", ip, port, streak)
+	} else {
+		message = fmt.Sprintf("Reply from %s (%s) on port %d TCP_conn=%d\n", hostname, ip, port, streak)
```

Comment:
Why did you remove the RTT?

Reply:
i'm sorry i must have missed that, fixing now